wgsl/index.bs

```diff
@@ -10894,6 +10961,12 @@ when [=uniformity analysis=] cannot prove that a particular [[#collective-operat
 * If a uniformity failure is triggered for a [[#sync-builtin-functions|synchronization builtin]],
     an [=severity/error=] [=diagnostic=] is [=triggered=],
     which results in a [=shader-creation error=].
+* If a uniformity failure is triggered for a [[#subgroup-builtin-functions|subgroup]] or
+    [[#quad-builtin-functions|quad]] builtin, then a [=trigger/subgroup_uniformity=]
+    [=diagnostic=] is [=triggered=]
+    * The diagnostic's [=diagnostic/triggering location=] is the location of the [=call site=] of that builtin.
```

Comment:
Again, need to mention the `subgroupShuffle*` delta value as the other possible triggering location?